conf/modules.config

```diff
@@ -469,10 +469,17 @@ process {
         ext.args = { "${meta.db_params}" }
         ext.prefix = params.perform_runmerging ? { "${meta.id}_${meta.db_name}.bracken" } : { "${meta.id}_${meta.run_accession}_${meta.db_name}.bracken" }
         publishDir = [
+	    [
             path: { "${params.outdir}/bracken/${meta.db_name}/" },
             mode: params.publish_dir_mode,
             pattern: '*.tsv'
-        ]
+            ],
+	    [
+            path: { "${params.outdir}/bracken/${meta.db_name}/" },
+            mode: params.publish_dir_mode,
+            pattern: '*bracken.kraken2.report_bracken*.txt'
```

Comment:
It's is not worth making it optional with a flag, it would be more concise to just update the glob in the `tsv` output e.g. with an OR statement

Reply:
Aha! I wasn't aware that was possible. Is the syntax simply pattern: 'x' OR 'y'?

Reply:
I guess perhaps you mean `'*{txt,tsv}'?`